src/frontend/src/optimizer/plan_node/logical_agg.rs

```diff
@@ -126,9 +138,9 @@ impl LogicalAgg {
         let in_pks = self.input().logical_pk().to_vec();
         let in_append_only = self.input().append_only();
         let in_dist_key = self.input().distribution().dist_column_indices().to_vec();
-        let get_merialized_input_state = |sort_keys: Vec<(OrderType, usize)>,
-                                          include_keys: Vec<usize>|
-         -> MaterializedAggInputState {
+        let get_materialized_input_state = |sort_keys: Vec<(OrderType, usize)>,
+                                            include_keys: Vec<usize>|
+         -> (TableCatalog, Vec<usize>) {
```

Comment:
May be better to still return `XxxState` as before? Given that the function name is `get_xxx_state`

Reply:
> May be better to still return `XxxState` as before? Given that the function name is `get_xxx_state`

fixed